k8s/test-env/kf-a.yml

```diff
@@ -55,7 +55,7 @@ spec:
     spec:
       containers:
         - name: ok
-          image: txn2/ok:3.0.0
+          image: c6o/ok:3.0.0
```

Comment:
I'm not really sure what this file is for, or what the txn2/ok docker container is, but c6o/ok doesn't exist, so not sure if we can just rename it.

Potential options:
1. Remove this file altogether.
2. Keep using txn2/ok
3. Find out where this docker container comes from, and publish our own @c6o/ok.